bluebottle/time_based/messages.py

```diff
@@ -101,11 +101,8 @@ class ParticipantAddedNotification(TransitionMessage):
     }
 
     def get_recipients(self):
-        """activity owner"""
-        if self.options.get('user') != self.obj.user:
-            return [self.obj.user]
-        else:
-            return []
+        """participant"""
```

Comment:
I am pretty sure we want this here. We should only send this message if someone else adds you to an activity.